packages/powerapps-lib/src/schema/processors/__tests__/example-src-object.js

```diff
@@ -356,7 +356,7 @@ export const applicationResponseObject = {
   sdds_organisationid: null,
   sdds_ecologistid: {
     '@odata.etag': 'W/"3285316"',
-    fullname: 'Mr Brian Ecologist',
+    lastname: 'Mr Brian Ecologist',
```

Comment:
Can we please update the value to better reflect the key?

Reply:
Names and title capture are currently under review. I am going to change this at the point there is an agreed way forward.

Reply:
That makes sense - thanks for explaining